core/websock.js

```diff
@@ -12,8 +12,7 @@
  * read binary data off of the receive queue.
  */
 
-import Util from "./util.js";
-
+import * as Log from './util/logging.js';
```

Comment:
We should remember to update websockify as well so we don't get out of sync again.